src/components/select/select.js

```diff
@@ -1276,7 +1276,7 @@ function SelectProvider($$interimElementProvider) {
      * Calculate the
      */
     function calculateMenuPositions(scope, element, opts) {
-      var 
+      var
```

Comment:
Can you make all those with var? or without it cause there are var deceleration right after..